src/pages/Search/SearchPageBottomTab.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {View} from 'react-native';
+import MenuItem from '@components/MenuItem';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useActiveRoute from '@hooks/useActiveRoute';
+import useSingleExecution from '@hooks/useSingleExecution';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as Expensicons from '@src/components/Icon/Expensicons';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type SearchMenuItem = {
+    title: string;
+    icon: IconAsset;
+    action: () => void;
+};
+
+function SearchPageBottomTab() {
```

Comment:
This component should be names `SearchFilters`

Reply:
@luacmartins since Im also working on this here is my understanding:

on small screens we plan to have:

`SearchPageBottomTab` (this is the parent of the whole page)
   - `SearchFilters` (I'm working on it right now https://github.com/Expensify/App/issues/39881)
   - (name to agree upon) `ListOfSearchResults`

My understanding is that when we divide it like this, then we can easily render the search results both on wide and small screens in a correct way. Filters always render inside bottom bar nav (left side) nav. Whereas the search results go either to `SearchPage` (if wide screen) or `SearchPageBottomTab` if small screen.

This name could also be changed to `SearchLHN` or similar if you prefer, but I think the `SearchFilters` should stay a child component of this one.